### DISCUSSION TITLE
Installing Bleak in PyPy

### DISCUSSION THREAD

**Discussion by ghost** [hbldh/bleak#1179] [Q&A]
I cannot install latest bleak version in a PyPy virtual environment. How do I do it? Please help.

**Comment by dlech** [comment]
Bleak requires binary dependencies on Windows and Mac in order to access the OS Bluetooth APIs, so it probably isn't going to work with PyPy. Should be possible on Linux though since the D-Bus library is pure Python.

**Reply by ghost** [reply]
Thanks 🙏
